### PR TITLE
addressed issues with webpack config & added SCSS file

### PR DESCRIPTION
**Pull request by KateFarrow**
Webpack was required as a dev dependency but oddly not installed - this has now been addressed and corrected. 

I created a 'dist' file for my vendor.js with a view towards best practice in project file naming conventions.

Lastly I created a SCSS file from my CSS just to be a bit fancy.

### CODE REVIEW

File: dist/app.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (rnewstead1) [inline dist/app.js]: I think these files in `dist` should not be commited to git either (as they are generated by webpack). So you should add them to `.gitignore`.

Comment (KateFarrow) [reply dist/app.js]: Ah ok, have added the dist file to the gitignore file and committed the update

Comment (rnewstead1) [reply dist/app.js]: You should also `git rm -r` the files currently in `dist`!

**Comment by KateFarrow** [comment]
Dist file and randomly generated vendor.js (in wrong folder) removed as per Rachel's wise advice! Thank you :)

**Comment by rnewstead1** [comment]
LGTM 🎉

**Pull request merged**